### PR TITLE
create drive.md

### PR DESCRIPTION
**Pull request by nickyvm**
Added details about new drive settings.

### CODE REVIEW

**Review by natefreng — COMMENTED** [review]
Good by me

File: content/departments/tech-ops/tools/drive.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+If you have a set of documents to be shared publicly, Tech Ops will create a drive where the files and folders have the ability to “share with anyone with a link.” Externally shared folders will have the title "EXT - Sourcegraph - XXX".
+
+## Collaborating with people outside of Sourcegraph
+
+Collaborating on files with people outside of Sourcegraph is available to everyone through the use of allowlists. Any folder or file can be shared with people at an organization whose domain has been added to our allowlist. After adding an email account, Drive will either allow the invite to be sent (meaning the domain is already on the allowlist) or else Drive will send a warning stating that this domain is not on our allowlist.
+
+To request a domain be added, please post in #it-tech-ops. Domains will be reviewed periodically to ensure they are still active collaborators.
+
+## Using google service accounts
```
Comment (milespatton) [inline content/departments/tech-ops/tools/drive.md:33]: Don't mean to nitpick but Google should prob be capitalized

File: content/departments/tech-ops/tools/drive.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Sharing permissions
+
+There are several ways to share documents within Google Drive.
+
+- **Restricted**: Only people specifically invited to the file can access that file. Collaborators can include anyone with an @sourcegraph email address or a user from an allowlisted domain. This setting is standard to all Sourcegraph accounts.
+- **Sourcegraph**: Anyone with an @sourcegraph email can access the document with the link to the file. Collaborators can include anyone with an @sourcegraph email address or any user from an allowlisted domain. This setting is standard to all Sourcegraph accounts.
+- **Anyone with the link** : Anyone with the link can access the document. This should be restricted to only files that can be shared publicly and do not contain any sensitive teammate or client information. This permission is limited to our publicly shared folders only.
+
+From each of these sharing levels you can set how you want collaborates to interact with the file (edit, view, or comment). More information about getting started with Google Drive can be found [here.](https://support.google.com/a/users/answer/9310248?hl=en)
```
Comment (milespatton) [inline content/departments/tech-ops/tools/drive.md:13]: I think there's a typo here --> "...you can set how you want collaborators to interact..."

**Review by milespatton — COMMENTED** [review]
Looks great to me - just 2 very minor grammatical things

**Pull request merged**